### ISSUE TITLE
feature request: show EPSG Code behind layer name

### ISSUE THREAD

**Issue by Heckantrieb** [qgis/QGIS#40557]
I have a feature request:

I would like to see an option to show the EPSG Code of any layers in the layers panel.

**Comment by Gustry** [comment]
As a workaround, you can have a look at the "LayerBoard" plugin. It will display all layers with their EPSG code, like a dashboard.

Note that only a subset has been ported to QGIS 3

**Comment by Heckantrieb** [comment]
> As a workaround, you can have a look at the "LayerBoard" plugin. It will display all layers with their EPSG code, like a dashboard.

Thanks! Just added "Layer Board". Anyway an option to see the CRS of every layer in the layers panel would be very helpful and a lot more comfortable than using a plugin.

**Comment by TurboGraphxBeige** [comment]
How would you imagine the way it's presented? Something similar to the "Show Feature Count"? A toggle in the context menu and a something like this?
[ESPG:4326] My layer name

**Comment by Heckantrieb** [comment]
Yes, similar to "Show Feature Count" would be perfect! Just like the number of features, the EPSG code would be shown behind the layer name in the layers panel in squared brackets:
"My layer name [ESPG:4326]"
